src/common/comp/brute_force.cc

```diff
@@ -117,8 +118,10 @@ BruteForce::Search(const DataSetPtr base_dataset, const DataSetPtr query_dataset
                                            cur_distances, cur_labels, bitset);
                     break;
                 }
-                default:
+                default: {
+                    LOG_KNOWHERE_ERROR_ << "Invalid metric type [" << cfg.metric_type << "].";
```

Comment:
suggest to change like this:
LOG_KNOWHERE_ERROR_ << "Invalid metric type: " << cfg.metric_type;